bin/qserv-test-integration.py

```diff
@@ -49,30 +49,52 @@
 from lsst.qserv.admin import commons
 from lsst.qserv.admin import logger
 from lsst.qserv.tests import benchmark
-from lsst.qserv.tests.unittest.testIntegration import suite
+from lsst.qserv.tests.unittest import testIntegration, testCall
 
 # ---------------------------------
 # Local non-exported definitions --
 # ---------------------------------
 _LOG = logging.getLogger()
 
 
+# To add test suites: add a name (key) and a factory lambda function (value) to the all_tests dictionary.
+# Then it will be available in the arguments to the script, and will be run by default if test names aren't
+# explicitly passed in by the caller.
+all_tests = {'testCall': lambda : testCall.suite(),
+             'testIntegration': lambda: testIntegration.suite(multi_node)}
+
+
 def _parse_args():
 
     parser = argparse.ArgumentParser(
-        description='''Qserv integration tests suite. Relies on python unit
-testing framework, provide test meta-data which can be used for example in a
-continuous integration framework or by a cluster management tool. Configuration values
-are read from ~/.lsst/qserv.conf.''',
+        description=("Qserv integration tests suite. Relies on python unit testing framework, provide test "
+                     "meta-data which can be used for example in a continuous integration framework or by a "
+                     "cluster management tool. Configuration values are read from ~/.lsst/qserv.conf."),
         formatter_class=argparse.ArgumentDefaultsHelpFormatter
     )
 
+    parser.add_argument('-t', '--run-tests',
+        help="The name of one or more test suites to execute. May include one or more of {}".format(
+             list(all_tests.keys())),
```

Comment:
You can probably replace `{}` with `%(choices)s` (and drop `format()`)

Reply:
I thought `format` was the pythonic way?

Reply:
Indeed `format` or f-string is the default choice unless we have a better mechanism. For `logging` for example we don't do `format` to avoid unnecessary formatting. Here there is also a good reason not to use format - you have already passes the list of choices to `choices` argument, so to avoid repeating yourself (which would have maintenance cost) you can use `%(choices)s`. in the help string (and also like in `logging` case save few CPU cycles by not pre-formatting that string)